lib/fog/core/scp.rb

```diff
@@ -1,43 +1,5 @@
 module Fog
   module SCP
-    def self.new(address, username, options = {})
-      if Fog.mocking?
-        Fog::SCP::Mock.new(address, username, options)
-      else
-        Fog::SCP::Real.new(address, username, options)
-      end
-    end
```

Comment:
I think we do need that on the fog-mock side to be able to create Mock or Real classes when testing but not on the fog-core side where only Real objects should be living.